app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  skip_before_action :verify_authenticity_token
```

Comment:
This will make every request to the application controller skip user verification, do we want to add it?

Reply:
I uploaded by mistake